src/components/SelectCountry/index.tsx

```diff
@@ -0,0 +1,120 @@
+import React, { useState } from "react";
+import CountryDropDown from "./CountryDropDown";
+import styled from "styled-components";
+
+function SelectCountry() {
+  const [showDropDown, setShowDropDown] = useState<boolean>(false);
+  const [selectCountry, setSelectCountry] = useState<string>("");
+  const countries = () => {
+    return ["KR, Korea", "US, United States", "CN, China", "JP, Japan"];
+  };
+
+  const toggleDropDown = () => {
+    setShowDropDown(!showDropDown);
+  };
+
+  const outFoucsHandler = (e: React.FocusEvent<HTMLButtonElement>) => {
```

Comment:
```suggestion
import { FocusEvent } from "react";

  const outFoucsHandler = (e: FocusEvent<HTMLButtonElement>) => {
```

로 아래도 다 통일하는 게 어떤가요 ?? 저는 다 이렇게 해왔어서 맞추면 좋을 것 같아요 ~!

Reply:
넵 참고해서 다시 리퀘 할게요 !